Makefile

```diff
@@ -9,9 +9,9 @@ S3_TARGET?=s3://public-repo-1.hortonworks.com/HDP/cloudbreak/
 #S3_TARGET=s3://public-repo.sequenceiq.com
 
 ifeq ($(GIT_BRANCH), release)
-FLAGS="-X main.Version $(VERSION)"
+FLAGS="-X main.Version=$(VERSION)"
```

Comment:
"link: warning: option -X main.Version 0.1 may not work in future releases; use -X main.Version=0.1" sometimes we have to fix it as all other member in the community :)